lib/neek.js

```diff
@@ -7,22 +8,22 @@ module.exports = {
 
   unique: function unique(input, output, callback) {
 
-    if (!isReadStream(input)) {
+    if (!isReadStream(input) && !isString(input)) {
       throw new Error('No input stream specified!');
     }
 
-    if (!isWriteStream(output)) {
+    if (!isWriteStream(output) && !isString(output)) {
       throw new Error('No output stream specified!');
     }
 
-    if (typeof input === 'string') {
+    if (isString(input)) {
```

Comment:
Weird, the review seems to have swallowed a comment.

Can you please move this check and the one for a String output prior to L11 and L15 respectively? That way you can also cut out those `!isString` calls in the conditions.